models/pytorch2onnx/ort_run_frozen.py

```diff
@@ -82,7 +82,7 @@ def get_numpy_dtype(onnx_dtype):
 for k, v in args.symbolic_dims.items():
     sess_options.add_free_dimension_override_by_name(k, int(v))
 
-ort_session = ort.InferenceSession(args.file, sess_options)
+ort_session = ort.InferenceSession(args.file, sess_options, providers=['CUDAExecutionProvider', 'CPUExecutionProvider'])
```

Comment:
It would raise error if no cuda device. 

Suggest to detect fp16 then set cuda provider in nnfusion.